gbn/gbn_conn.go

```diff
@@ -556,8 +542,9 @@ func (g *GoBackNConn) receivePacketsForever() error { // nolint:gocyclo
 				// the resend, and therefore won't react to the
 				// NACK we send here in time.
 				sinceSent := time.Since(lastNackTime)
-				recentlySent := sinceSent <
-					g.cfg.resendTimeout*2 //nolint:gomnd
+
+				timeout := g.timeoutManager.GetResendTimeout()
+				recentlySent := sinceSent < timeout*2 //nolint:gomnd
```

Comment:
Hmm, should we just disable the `gomnd` linter? Since normally we're pretty good with not using magic numbers anyway.

Reply:
Thanks! I agree, so I added that disable to #87!